PagedRouteCollection.php

```diff
@@ -92,7 +85,7 @@ public function next()
         if (false === $result) {
             $this->loadNextElements($this->current + 1);
         }
-        $this->current++;
+        ++$this->current;
```

Comment:
wtf but whatever. it does exactly the same here, as we don't do anything else than increase the value.